h/views/client.py

```diff
@@ -49,6 +49,9 @@ def sidebar_app(request, extra=None):
     app_config = {
         'apiUrl': request.route_url('api.index'),
         'authDomain': request.authority,
+
+        # OAuth config.
+        'oauthAuthorizeUrl': request.route_url('oauth_authorize'),
```

Comment:
I wonder if this should go in [the links endpoint](https://github.com/hypothesis/h/blob/master/h/views/api.py#L153)? https://hypothes.is/api/links